### ISSUE TITLE
Jump around two opened windows

### ISSUE THREAD

**Issue by RichardIvan** [DavidLGoldberg/jumpy#99]
Thank you for this amazing package!

I was wondering if you think it'd be possible to add the above.

Nuclide has a feature to Open/Close Working Sets. This works across all opened windows so I'm thinking it cold.

**Comment by DavidLGoldberg** [comment]
Thanks for the love.  I actually had an issue for this open:
#48 

I actually just commented on it the other day, I had some ideas, but I'll definitely check out the above.  Glad to know they found *some* way to do it.  I left a comment in the other issue that apparently the floobits guys got it to work as well.  I unfortunately haven't spent any time towards it.  I think some of the refactoring I'm doing (still in progress) and the work that was done in a recent pr #93 should facilitate this a lot.

Basically, I think Treeview, tabs selection and windows should come all relatively soon.  Just a hobby/side project of course though, so soon, is probably a month or so :\

Closing, because this should be referenced from #48
